cfme/networks/floating_ips.py

```diff
@@ -53,6 +56,36 @@ def all(self):
                 floating_ips.append(self.instantiate(address=ip.data['address']))
         return floating_ips
 
+    def create(self, tenant, provider, network_manager, network_name,
+               floating_ip_address=None, fixed_ip_address=None, network_port_id=None):
+        """Create subnet
+
+        Args:
+            tenant: (str) name of the tenant to place FIP to
+            provider: crud object of Openstack cloud provider
+            network_manager: (str) name of network manager
+            network_name: (str) name of the network to create FIP under
+            floating_ip_address: (str) Floating Address(Name) of FIP, for example: 192.168.12.2/24
+            fixed_ip_address: (str) Fixed Address(Name) of FIP, for example: 192.168.12.2/24
+            network_port_id: (str) Id of network port to associate FIP with
+
+        Returns: instance of cfme.networks.floating_ips.FloatingIp
+        """
+        view = navigate_to(self, 'Add')
+        view.fill({'network_manager': network_manager,
+                   'network': network_name,
+                   'network_port_id': network_port_id,
+                   'floating_ip_address': floating_ip_address,
+                   'fixed_ip_address': fixed_ip_address,
+                   'cloud_tenant': tenant})
+        view.add.click()
+        view.flash.assert_success_message(f'Floating IP "{floating_ip_address}" created')
```

Comment:
We want to `assert_no error()` here instead and have the actual flash message asserted in a test.

Reply:
Agreed @jawatts, @mmojzis can you cover this flash assertion with a positive CRUD test case? Looks like only negative test cases are included in this PR.